src/Compilers/Core/Portable/SourceGeneration/ArtifactProducerAttribute.cs

```diff
@@ -33,7 +33,7 @@ public ArtifactProducerAttribute(string firstLanguage, params string[] additiona
             var languages = new string[additionalLanguages.Length + 1];
             languages[0] = firstLanguage;
             for (int index = 0; index < additionalLanguages.Length; index++)
-                languages[index + 1] = additionalLanguages[index];
+                Array.Copy(additionalLanguages, sourceIndex: 0, languages, destinationIndex: 1, additionalLanguages.Length);
```

Comment:
The point was to replace the entire for loop with an Array.Copy :)

Reply:
D'oh!